x/gamm/keeper/keeper.go

```diff
@@ -30,6 +30,7 @@ type Keeper struct {
 	accountKeeper       types.AccountKeeper
 	bankKeeper          types.BankKeeper
 	communityPoolKeeper types.CommunityPoolKeeper
+	swaprouterKeeper    types.SwaprouterKeeper
 }
 
 func NewKeeper(cdc codec.BinaryCodec, storeKey sdk.StoreKey, accountKeeper types.AccountKeeper, bankKeeper types.BankKeeper, communityPoolKeeper types.CommunityPoolKeeper) Keeper {
```

Comment:
don't we need to add to the return statement:

`swaprouterKeeper: swaprouterKeeper,`

Reply:
No, because swaprouter keeper is being set by the `SetSwapRouterKeeper` function due to the ordering of keeper initialization in `app/keepers`